src/main/java/org/mule/module/apikit/validation/body/form/MultipartFormValidator.java

```diff
@@ -45,20 +46,18 @@ public TypedValue validate(TypedValue originalPayload) throws InvalidFormParamet
 
       IParameter expected = formParameters.get(expectedKey).get(0);
```

Comment:
if (formParameters.get(expectedKey).size() != 1) {
        //do not perform validation when multi-type parameters are used
        continue;
      }

      IParameter expected = formParameters.get(expectedKey).get(0);

Replace with --------->

List<IParameter> params = formParameters.get(expectedKey)
if (params != null && params.size() == 1)